packages/SwingSet/src/vats/comms/state.js

```diff
@@ -244,12 +244,20 @@ export function makeStateKit(state) {
     assert(!p.resolved);
     insistCapData(resolution.data);
     p.resolved = true;
+    p.kernelAwaitingResolve = true;
```

Comment:
let's add a TODO note here, I'm 80% convinced that `p.kernelAwaitingResolve` will always have the same value as `p.kernelIsSubscribed`, and it might pay off to prove/disprove that claim and collapse the flags if true